htmlbars-version/app/components/focusing-outlet.js

```diff
@@ -0,0 +1,19 @@
+import Ember from "ember";
+import FocusingOutletMixin from 'ember-a11y/mixins/focusing';
+import getOwner from 'ember-getowner-polyfill';
+
+const {
+  get,
+  Component
+} = Ember;
+
+let FocusingOutlet = Component.extend(FocusingOutletMixin, {
+  getOwner,
```

Comment:
I'm not sure if this is safe or what it closes over. @rwjblue can you quickly confirm or deny?